public/src/js/components/StripeCheckoutComponent.js

```diff
@@ -21,20 +21,20 @@ module.exports = React.createBackboneClass({
 
   render: function() {
 
-    var disabled = isNaN(this.getModel().get('paymentAmount')) || this.getModel().get('paymentAmount')<= 0; 
+    var disabled = isNaN(this.getModel().get('paymentAmount')) || this.getModel().get('paymentAmount')<= 0;
 
     return (
       <StripeCheckout
         token={this.onToken}
-        stripeKey={process.env.STRIPE_PUBLISHABLE_KEY}
+        stripeKey="pk_test_LIWkhhJVRCPoGBO3SXaUXmTS"
```

Comment:
And also, if you accidently commit a key locally, it is not enough to remove it and commit again, as the key is now stored in the commit history, you either have to rebase, or undo your last commit(s) to go back in time and recommit without the keys.